services/email-service/handler/email.handler.ts

```diff
@@ -0,0 +1,86 @@
+import fs from 'fs';
+import FormData from 'form-data';
+import Mailgun from 'mailgun.js';
+import { Resend } from 'resend';
+import { MailtrapClient } from "mailtrap"
+import { serviceConfig } from '../configs/config';
+
+const mailgunSdk = new Mailgun(FormData);
+const mailtrapSdk = new MailtrapClient({ token: serviceConfig.mailtrap.apiKey });
+const resendSdk = new Resend(serviceConfig.resend.apiKey);
+
+export class EmailHandler {
+    private mailgun: any;
```

Comment:
does `Mailgun` not providing their own class/type for this? we need to avoid the usage of `any`

Reply:
it should be like this : 
```ts
private mailgun: Mailgun | null;
private mailgunClient: IMailgunClient | null;
````
just pass null on the constructor so it can be conditionally used